redaxo/src/addons/install/lib/rights.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * @package redaxo\install
+ *
+ * @internal
+ */
+class rex_set_rights
```

Comment:
Für diese logik/klasse müssen wir noch einen besseren namen/platz finden.
Aber erstmal müssen wir gegenchecken ob es das gemeldete problem tatsächlich löst

Reply:
in `rex_file` unterbringen?

Reply:
Eher `rex_dir`, oder?

Reply:
Oder halt erstmal nur direkt an der einen Stelle, wo es verwendet wird.